src/main/java/nextstep/subway/domain/Section.java

```diff
@@ -0,0 +1,56 @@
+package nextstep.subway.domain;
+
+import javax.persistence.*;
+
+@Entity
+public class Section {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne(cascade = CascadeType.PERSIST)
+    @JoinColumn(name = "line_id")
+    private Line line;
+
+    @ManyToOne(cascade = CascadeType.PERSIST)
+    @JoinColumn(name = "up_station_id")
+    private Station upStation;
+
+    @ManyToOne(cascade = CascadeType.PERSIST)
+    @JoinColumn(name = "down_station_id")
+    private Station downStation;
+    private int distance;
+
+    public Section() {
+    }
+
+    public Section(Station upStation, Station downStation, int distance) {
+        this.upStation = upStation;
+        this.downStation = downStation;
+        this.distance = distance;
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setLine(Line line) {
+        this.line = line;
+    }
+
+    public Station getUpStation() {
+        return upStation;
+    }
+
+    public Station getDownStation() {
+        return downStation;
+    }
+
+    public boolean isConnectable(Section section) {
+        return section.getUpStation().equals(downStation);
+    }

```

Comment:
```suggestion
   public boolean isConnectable(Section section) {
        return section.isEqualToUpStation(downStation);
    }

    private boolean isEqualToUpStation(Station station){
        return this.upStation.equals(station);
    }
```

>어떻게 하면 도메인에 로직을 옮길 수 있는지를 고민하면서 작성했습니다. 최대한 getter, setter를 안 쓰고 싶었는데, 일부는 아쉽네요.

getter와 setter를 전혀 안쓰기는 어려운 일 인 것 같아요 ㅜ_ㅜ section의 값을 getter를 사용하지 않고 section에게 메세지를 보내서 확인할 수 있도록 잘 해주신 것 같아요 👍  이 부분이 맘에 안드셨다면 이렇게도 작성해볼 수 있을 것 같아요~!

Reply:
일관성 때문에 고민했던 부분인데, 이렇게 하니 더 좋네요!
3fe874e